addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/Pair.java

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Could you use a Pair class from apache commons or simply AbstractMap.SimpleEntry<K,V>? This would avoid reimplementing classes that are already implemented.

Reply:
lol - don't even use the class (and not sure why I defined it - might have been a leftover from some time)